modules/microservice/main.tf

```diff
@@ -465,6 +486,19 @@ resource "azurerm_function_app" "microservice" {
     type         = local.appservice_identity_type
     identity_ids = local.user_assigned_identity_ids
   }
+
+  auth_settings {
```

Comment:
I believe this should be a dynamic block

Reply:
Good call out, moved to dynamic